File: mealpository_backend/database/instruction.py
class Instruction:
    def __init__(self, 
                 instruction_id, number, instruction, recipe_id
                ):
        self.instruction_id = instruction_id
        self.number = number
        self.instruction = instruction
        self.recipe_id = recipe_id

    def serialize_self(self):
        return {
            'instruction_id': self.instruction_id,
            'number': self.number,
            'instruction': self.instruction,
            'recipe_id': self.recipe_id

        }

File: mealpository-frontend/src/app/models/instruction.ts
export class Instruction {
    
    instruction_id: string;
    number: string;
    instruction: string;
    recipe_id: string;
    

    constructor (
        instruction_id: string,
        number: string,
        instruction: string,
        recipe_id: string

    ) {
        this.instruction_id = instruction_id;
        this.number = number;
        this.instruction = instruction;
        this.recipe_id = recipe_id;
    }

}